src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -153,6 +159,24 @@ export default function FundingForm({ onFinish }: { onFinish: VoidFunction }) {
       )
       dispatch(editingV2ProjectActions.setDuration(durationInSeconds ?? '0'))
 
+      // reset discount rate if duration is 0
+      if (!durationInSeconds || durationInSeconds === '0') {
```

Comment:
if someone goes to token tab, then back to funding tab and makes changes, then we might need to reset some values. we do that below

Reply:
Yeah got it